packages/xsnap/build.env

```diff
@@ -0,0 +1,4 @@
+MODDABLE_URL=https://github.com/agoric-labs/moddable.git
```

Comment:
I have seen this file show up dirty on my environment, and I think is a consequence of https://github.com/Agoric/agoric-sdk/pull/7048/files#r1117951126

But here I suppose you're taking advantage of this to rebuild?

Reply:
This issue should now be fixed.  `build.env` is checked into Git for this PR.